book/content/remote_collaboration/resources/resources.md

```diff
@@ -1,18 +1,78 @@
 ## Managing and sharing resources across different stakeholders when collaborating remotely
 
-<!---Add a few sentences capturing the concepts explained below--->
+Since we have gained an insight about setting [**Guidelines to ensure that remote collaborations are successful**](https://deploy-preview-962--the-turing-way.netlify.app/remote_collaboration/protocols/protocols.html#Tools-for-communication-and-collaboration), let us have a look into how to manage and share the available resources in a remote collaboration. 
+This step results in an efficient distribution of the resources and the maximum utilisation of the remotely collaborating members.
 
-### Content
-<!---Add sub-heading--->
+### Stakeholders
+---
+- **Stakeholders** refer to individuals or group of individuals who might affect or be affected by the decisions, actions or policies of an organization.  
+The distribution of the resources takes into account the different stakeholders, and their stakes. 
+The best practice is to distribute the resources in proportionate to the stakes, various stakeholders hold.
+
+- The stakeholders can be categorized mainly into:
+	1. **Internal Stakeholders** which includes the individuals who have collaborated for the project, including the project manager and also the leaders of smaller teams.
+	2. **External Stakeholders** which includes the customers, the testers, partners, vendors.
```

Comment:
I personally don't think that this category works for academic or data science project. As mentioned in the article you referred, maybe you can use the same approach to define different stakeholders for the data science, what each of their roles are and what resources they need.

Reply:
Ok sure @malvikasharan. I will look into the definitions as per a data science project.